### PR TITLE
Refresh generated proto classes

### PR DESCRIPTION
**Pull request by vam-google**
This does not include namespace change yet

### CODE REVIEW

File: src/LongRunning/Gapic/OperationsGapicClient.php [hunk 1]
```diff
@@ -48,8 +48,10 @@
 use Google\GAX\AgentHeaderDescriptor;
 use Google\GAX\ApiCallable;
 use Google\GAX\CallSettings;
+use Google\GAX\GrpcConstants;
```
Comment (jdpedrie) [inline src/LongRunning/Gapic/OperationsGapicClient.php]: This isn't used (and doesn't exist anymore).

File: src/LongRunning/Gapic/OperationsGapicClient.php [hunk 2]
```diff
@@ -48,8 +48,10 @@
...
 use Google\GAX\GrpcCredentialsHelper;
 use Google\GAX\PageStreamingDescriptor;
+use Google\GAX\PathTemplate;
```
Comment (jdpedrie) [inline src/LongRunning/Gapic/OperationsGapicClient.php]: This is also not used in this file.

File: src/LongRunning/Gapic/OperationsGapicClient.php [hunk 3]
```diff
@@ -95,6 +97,7 @@
  */
 class OperationsGapicClient
 {
+
```
Comment (dwsupplee) [inline src/LongRunning/Gapic/OperationsGapicClient.php]: https://github.com/GoogleCloudPlatform/google-cloud-php/pull/715#discussion_r147433751

File: src/LongRunning/Gapic/OperationsGapicClient.php (hunk 2 above)
Comment (vam-google) [reply src/LongRunning/Gapic/OperationsGapicClient.php]: fixing cs php-cs-fixer in artman fixed the problem ([artman PR](https://github.com/googleapis/artman/pull/310))

File: src/LongRunning/Gapic/OperationsGapicClient.php (hunk 3 above)
Comment (vam-google) [reply src/LongRunning/Gapic/OperationsGapicClient.php]: fixing cs php-cs-fixer in artman fixed the problem ([artman PR](https://github.com/googleapis/artman/pull/310))

**Pull request merged**